doc/releases/changelog-dev.md

```diff
@@ -84,6 +87,7 @@
 This release contains contributions from (in alphabetical order):
 
 Abhishek Abhishek,
+Gabriel Bottrill,
```

Comment:
```
The sphinxcontrib.applehelp extension used by this project needs at least Sphinx v5.0; it therefore cannot be built with this version.
Traceback (most recent call last):
  File "/entrypoint.py", line 22, in <module>
    action.build_all_docs(github_env, [os.environ.get("INPUT_DOCS-FOLDER")])
  File "/sphinx_action/action.py", line 167, in build_all_docs
    raise RuntimeError("Build failed")
RuntimeError: Build failed
```

I thought it was me forgetting a comma that made the docs test fail, but it doesn't seem related to my changes. Anything I need to do?

Reply:
It's an old dependency causing problems in multiple places. A PR to fix this has been merged to `master`, so whenever you update the branch this failure should go away

Reply:
thanks @mudit2812